style.css

```diff
@@ -1,8 +1,263 @@
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
 }
+
+html{
+  font-size: 16px;
+}
+
+body{
+  background-color: rgb(255, 255, 255);
+}
+
+.container{
+  width: 100%;
+  height: 100vh;
+  position: relative;
+}
+
+.flex{
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 0 1%;
+}
+
+.navigation{
+  width: 100%;
+  background-color: rgb(0, 0, 0);
+}
+
+.logo{
+  width: 15%;
+}
+
+.logo img{
+  width: 140px;
+  padding: 3%;
+  margin-top: 1%;
+  cursor: pointer;
+}
+
+.menu{
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+}
+
+.right-nav{
+  width: 30%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  
+}
+
+.navbar{
+  width: 85%;
+}
+
+.left-nav{
+  width: 70%;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+ul li{
+  color: rgb(255, 255, 255);
+  list-style: none;
+  font-size: 1rem;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.navbar a:hover {
+  text-decoration: underline;
+  text-decoration-color: rgb(255, 0, 0);
+}
+
+.search{
+  background-color: transparent;
+  border: 2px solid rgb(218, 20, 20);
+  border-radius: 8px;
+  color: rgb(255, 255, 255);
+  padding: 5px;
+  margin: 10px;
+  font-size: 1rem;
+  font-family: inherit;
+}
+
+.search:focus{
+  outline: 0;
+  background-color: transparent;
+}
+
+.search::placeholder{
+  color: rgb(255, 255, 255);
+}
+
+.togglebtn{
+  display: none;
+  font-size: 30px;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  color: #0094da;
+}
+
+#main{
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.movie {
+  width: 300px;
+  margin: 10px;
+  border-radius: 3px;
+  box-shadow: 0.2px 4px 5px rgba(0,0,0,0.1);
+  background-color: #fff;
+  position:relative;
+  overflow: hidden;
+}
+
+
+.movie img{
+  width:100%;
+}
+
+.movie-info{
+  color: black;
```

Comment:
Here you are using named color values. Please prefer to use one color unit and use it everywhere.